package.json

```diff
@@ -17,14 +17,16 @@
     "test": "yarn run lint && jest",
     "watch": "jest --watch",
     "lint": "eslint .",
-    "flow": "flow check",
     "fix": "eslint . --fix",
     "promote": "heroku pipelines:promote -r staging",
     "dump-schema": "babel-node ./scripts/dump-schema.js",
-    "heroku-postbuild": "babel index.js config.js -s inline -d build && babel lib -s inline -d build/lib && babel schema -s inline -d build/schema",
-    "precommit": "lint-staged"
+    "heroku-postbuild":
+      "babel index.js config.js -s inline -d build && babel lib -s inline -d build/lib && babel schema -s inline -d build/schema",
+    "precommit": "lint-staged",
+    "prettier-project": "prettier --write '**/*.js'",
+    "type-check": "tsc --noEmit --pretty"
```

Comment:
This is the same command as in Emission, and I added a `prettier-project` so that any style changes can be easily updated.